deepmd/utils/argcheck.py

```diff
@@ -464,6 +465,7 @@ def model_args ():
                   [Argument("type_map", list, optional = True, doc = doc_type_map),
                    Argument("data_stat_nbatch", int, optional = True, default = 10, doc = doc_data_stat_nbatch),
                    Argument("data_stat_protect", float, optional = True, default = 1e-2, doc = doc_data_stat_protect),
+                   Argument("data_bias_ntest", int, optional=True, default=10, doc=doc_data_bias_ntest),
```

Comment:
Do you compute the bias from the training data? 
"ntest" gives the impression of computing the bias from a test dataset, which is misleading.